File: Backend/PlanNGoUMS/src/main/java/com/planNGo/ums/service/OrganizerService.java
package com.planNGo.ums.service;

import java.io.IOException;

import org.springframework.web.multipart.MultipartFile;

import com.planNGo.ums.dtos.ApiResponse;
import com.planNGo.ums.dtos.OrganizerResp;
import com.planNGo.ums.dtos.UpdateOrganizer;

public interface OrganizerService {
//get all users
	//List<OrganizerDTO> getAllOrganizers();
	
	

	ApiResponse deleteOrganizer(Long organizerId);

	OrganizerResp getOrganizerDetails(Long organizerId);

	ApiResponse updateDetails(Long id, UpdateOrganizer organizer);

	

	ApiResponse uploadDocuments(Long userId, MultipartFile files, String docType);

	
	
	
	
}

File: Frontend/src/services/api/organizerService.js
import axios from 'axios';

export const organizerService = {
  uploadDocuments: async (userId, files, docTypes) => {
    const formData = new FormData();
    
    // Append files
    files.forEach(file => {
      formData.append('files', file);
    });
    
    // Append document types
    docTypes.forEach(docType => {
      formData.append('docType', docType);
    });
    
    return await axios.post(`http://localhost:9090/organizer/documents/${userId}`, formData);
  }
};